### DISCUSSION TITLE
WakaTime

### DISCUSSION THREAD

**Discussion by AssassinAguilar** [anuraghazra/github-readme-stats#1092] [Q&A] [answered]
I just made my wakatime account 2-3 days ago. So Did I have to wait for 7 days to get some data in the wakatime stats ? As now my readme Wakatime data shows no coding activity this week

**Comment by florianbussmann** [comment, marked as answer]
Checking your profile at https://wakatime.com/@AssassinAguilar it says

> This user has no coding activity, or has **privacy settings that prevent showing it**.

As you are wondering why your coding activity does not show up I guess you just have to change your privacy settings. In order to change these go to https://wakatime.com/settings/profile and check the following options
- [x] Display code time publicly
- [x] Display languages, editors, os, categories publicly

**Reply by AssassinAguilar** [reply]
Ok Thanks Dude :)

**Reply by michaelschwob** [reply]
I have both of these selected, yet I still do not see the WakaTime statistics showing up.

**Reply by UnknownSuperficialNight** [reply]
Same i have both selected but nothing shows up

**Reply by florianbussmann** [reply]
> Same i have both selected but nothing shows up

@UnknownSuperficialNight Your card looks good to me, did it resolve or still anything odd on your end?

![image](https://github.com/anuraghazra/github-readme-stats/assets/3610244/7a8d34e6-6292-422e-b7c0-22ba70ddb8f4)

**Reply by qwerty541** [reply]
Hey, @UnknownSuperficialNight! Acording to my experience with wakatime, it updates the data provided by API and shown in the graph on the profile page just once per day. Since you are new user, most likely there was just no data to show. At the moment looks like all works fine.

**Reply by JovanniChen** [reply]
ok, its right.
